Automattic/jetpack
# Search: Apply design polish to the Customberg customization interface

#### Changes proposed in this Pull Request:
* Changes links from the Customizer to Customberg for Search configuration.

#### Jetpack product discussion
See pcNPJE-7J-p2.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Ensure you have a Jetpack Search subscription for your site.
* Navigate to the following four pages and ensure that they link to the new customization interface (Customberg):
  1. At-a-glance: `/wp-admin/admin.php?page=jetpack#/dashboard`
  2. My plan: `/wp-admin/admin.php?page=jetpack#/my-plan`
  3. Performance settings: `/wp-admin/admin.php?page=jetpack#/performance`
  4. New Jetpack Search page: `wp-admin/admin.php?page=jetpack-search&a8ctest`
* Ensure that the new customization interface works as expected: 
`/wp-admin/admin.php?page=jetpack-search-configure`.

#### Notes for design reviewers:
* The new Jetpack Search customization interface (dubbed Customberg) can be found at: 
`/wp-admin/admin.php?page=jetpack-search-configure`
* Customberg currently looks like the following:
> <img width="1789" alt="Screen Shot 2021-08-18 at 5 17 16 PM" src="https://user-images.githubusercontent.com/4044428/129984134-3b55c232-19b0-4a82-bac4-17dc0b629b45.png">
* This PR changes existing links for configuring Search to point to Customberg.
* Customberg is feature-complete compared to configuring Jetpack Search within the Customizer.
* Customberg automatically collapses the wp-admin sidebar upon initialization.
* Customberg only works for WordPress 5.8 and up. The page will redirect to the Customizer experience if the installed WordPress version is less than 5.8.
* It's not possible to directly edit the blocks in the search overlay sidebar in Customberg. One must navigate to the Widgets editor to do so.
